requirements-dev-frozen.txt

```diff
@@ -170,6 +170,7 @@ s3transfer==0.6.1
 selenium==4.3.0
 sentry-arroyo==2.14.23
 sentry-cli==2.16.0
+sentry-devenv==1.1.3
```

Comment:
Let's not put this in place yet while people are beta-testing.

Reply:
Don't we need it for the checks to work?

Reply:
i'm not sure, but i'd like to ship devenv separately and before we add any doctor stuff